### ISSUE TITLE
Graduate to become an Impact project 

### ISSUE THREAD

**Issue by UlisesGascon** [expressjs/discussions#281]
Based on recent discussions in Slack (cc: @expressjs/express-tc), it seems like we can start thinking about requesting to be considered for a graduation from At-Large stage to become Impact stage.

[Documentation](https://github.com/openjs-foundation/cross-project-council/blob/main/PROJECT_PROGRESSION.md#impact-stage)

I was checking the acceptance criteria and the examples, and I think that we meet most of the criteria (note that our LTS plan will be available soon).

As a next step, I will bring this topic to the next CPC meeting to clarify the process and practical details.

**Comment by crandmck** [comment]
I agree we meet the acceptance criteria for an Impact project. I think it's a reasonable idea.

**Comment by IamLizu** [comment]
Referencing https://github.com/openjs-foundation/cross-project-council/issues/1378

**Comment by wesleytodd** [comment]
As we can close this I think now as we are officially an impact project 🎉
